tests/json.py

```diff
@@ -182,6 +182,21 @@ def foo():
     assert response_dict["count"] == 3
 
 
+def test_stupeflask_response_with_paginate_headers(json_app, client):
+    encoder_rules.append((Cursor, lambda c: c.data))
+
+    @json_app.route("/")
+    @paginate(limit=1, skip=0, headers=True)
+    def foo():
+        return Cursor([1, 2, 3])
+
+    response = client.get("/")
+    assert response.status_code == 200
+
+    assert "X-Total-Count" in response.headers
+    assert "Link" in response.headers
```

Comment:
We should test the content here also.

Reply:
Resolving because https://github.com/numberly/flask-stupe/pull/27/files#r237106990 makes sense. See https://github.com/numberly/flask-stupe/pull/27/files#r237139744 though.